services/src/main/java/org/keycloak/protocol/oidc/OIDCConfigAttributes.java

```diff
@@ -0,0 +1,26 @@
+package org.keycloak.protocol.oidc;
+
+public interface OIDCConfigAttributes {
```

Comment:
Could this be turned into a class with `public static final String` fields instead?

Reply:
Why?